README.md

```diff
@@ -58,6 +58,27 @@ my-teradata-db-profile:
       logmech: LDAP
 ```
 
+### Logdata
+
+The logon mechanism for Teradata jobs that dbt executes can be configured with the `logdata` configuration in your Teradata profile. Addtional data like secure token, distinguished Name, or a domain/realm name can be set in your Teradata profile using `logdata`. `logdata` is not used with the TD2 mechanism.
```

Comment:
I think this paragraph should list what mechanisms support logadata: KRB5, ldap, jwt.

Reply:
Added with supported mechanisms.